arch/xtensa/src/esp32s3/hardware/esp32s3_system.h

```diff
@@ -0,0 +1,1757 @@
+/****************************************************************************
+ * arch/xtensa/src/esp32s3/hardware/esp32s3_system.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_SYSTEM_H
+#define __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_SYSTEM_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include "esp32s3_soc.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* SYSTEM_CORE_1_CONTROL_0_REG register
+ * Core0 control regiter 0
+ */
+
+#define SYSTEM_CORE_1_CONTROL_0_REG (DR_REG_SYSTEM_BASE + 0x0)
+
+/* SYSTEM_CONTROL_CORE_1_RESETING : R/W; bitpos: [2]; default: 1;
+ * Set 1 to let core1 reset
+ */
+
+#define SYSTEM_CONTROL_CORE_1_RESETING    (BIT(2))
```

Comment:
minor
```suggestion
#define SYSTEM_CONTROL_CORE_1_RESETING    BIT(2)
```
here and in other places